doc/u-boot-env.md

```diff
@@ -32,10 +32,15 @@ setenv ethaddr 12:34:56:78:9A:BC
 
 ## TFTP boot
 ```
+# ./build-domd/tmp/deploy/images/YOUR_BOARD/xen-uImage
 setenv tftp_xen_load tftp 0x48080000 xen
-setenv tftp_dtb_load 'tftp 0x48000000 xen.dtb; fdt addr 0x48000000; fdt resize; fdt mknode / boot_dev; fdt set /boot_dev device nfs'
+# ./build-domd/tmp/deploy/images/YOUR_BOARD/HASH-YOUR_BOARD*-xen-YOUR_BOARD.dtb
+setenv tftp_dtb_load 'tftp 0x48000000 xen.dtb; fdt addr 0x48000000; fdt resize; fdt mknode / boot_dev; fdt set /boot_dev device nfs; fdt set /boot_dev my_ip ${ipaddr}; fdt set /boot_dev nfs_server_ip ${serverip}; fdt set /boot_dev nfs_dir "/srv/domd-YOUR_BOARD";'
```

Comment:
It would be nice to add a paragraph that describes meaning of this options.

Reply:
done